server/constants/constants.go

```diff
@@ -0,0 +1,14 @@
+package constants
+
+const (
+	SERVER_ADDRESS string = ":8000"
+	NETWORK_TCP string = "tcp"
```

Comment:
Go doesn't use screaming snake case, not even for constants. You also are usually better off using untyped constants and letting the compiler figure it out. Also, there is  constant in `net` for `"tcp"`

```suggestion
	DefaultServerAddress = ":8000"
```

Reply:
`net` doesn't have a constant for`"tcp"`
Verified here: https://pkg.go.dev/net#pkg-constants